test/list_resources/test_integration_list_role_members.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+
+"""
+CLI List Integration tests
+
+This test file handles the main test flows for the list command
+"""
+from test.util import test_helpers as utils
+from test.util.test_infrastructure import integration_test
+from test.util.test_runners.integration_test_case import IntegrationTestCaseBase
+
+
+# Not coverage tested since integration tests doesn't run in
+# the same build step
+class CliIntegrationTestList(IntegrationTestCaseBase):  # pragma: no cover
+    def __init__(self, test_name, client_params=None, environment_params=None):
+        super(CliIntegrationTestList, self).__init__(test_name, client_params, environment_params)
+
+    # *************** HELPERS ***************
+
+    def setUp(self):
+        self.setup_cli_params({})
+        # Used to configure the CLI and login to run tests
+        utils.setup_cli(self)
+        self.invoke_cli(self.cli_auth_params,
+                        ['policy', 'replace', '-b', 'root', '-f',
+                         self.environment.path_provider.get_policy_path("list_role_members")])
+
+    # *************** TESTS ***************
+
+    @integration_test()
+    def test_list_members_of_single_role(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['list', '--members-of', 'group:devs'])
+        account = self.client_params.account
+        expected_result = "[\n" \
```

Comment:
The order in the response is not consistent.
It is better to compare it in unordered way, see `test_integration_list_permitted_roles.py`.
Example:
```python
    def test_list_permitted_roles_for_distinct_resource_without_kind_return_success(self):
        output = self.invoke_cli(self.cli_auth_params,
                                 ['list', '--permitted-roles', 'one/password', '--privilege', 'read'])
        json_result = json.loads(output)
        expected_result = [f"{self.client_params.account}:user:someuser", f"{self.client_params.account}:user:admin"]
        self.assertCountEqual(expected_result, json_result)
```

Reply:
It tests the exact response the way the client sees it when he executes the command

Reply:
changed the regex to handle any order